kernel/kernel-util/src/main/java/org/sakaiproject/util/EmailNotification.java

```diff
@@ -645,6 +645,24 @@ protected int getOption(User user, String notificationId, String resourceFilter,
 		String type = getType(resourceFilter);
 		if (type != null)
 		{
+            // First, check the overrides.
```

Comment:
Personally I prefer space to tabs, but having both in one file is even worse.

Previously we've not reformatted old files as you lose the history so I think that's a bigger discussion to be had.